src/app/clusters/operational-credentials-server/operational-credentials-server.cpp

```diff
@@ -282,7 +282,6 @@ CHIP_ERROR DeleteFabricFromTable(FabricIndex fabricIndex)
 
 void CleanupSessionsForFabric(SessionManager & sessionMgr, FabricIndex fabricIndex)
 {
-    InteractionModelEngine::GetInstance()->CloseTransactionsFromFabricIndex(fabricIndex);
```

Comment:
So what calls CloseTransactionsFromFabricIndex now?  I don't see it being called from the delegate callback in IM engine.  And if the delegate callback is doing its work now, should CloseTransactionsFromFabricIndex just go away?

Reply:
https://github.com/project-chip/connectedhomeip/blob/master/src/app/clusters/operational-credentials-server/operational-credentials-server.cpp#L325
https://github.com/project-chip/connectedhomeip/blob/master/src/app/clusters/operational-credentials-server/operational-credentials-server.cpp#L313

Based upon the above call flow, 
yes, i think CloseTransactionsFromFabricIndex method should be removed.